src/style.css

```
@@ -1 +1,38 @@
 /* add styles here */
+body{
+  margin: 0;
+  padding: auto;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(/src/images/star.svg);
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
```

```suggestion
  background: url(/src/images/star.svg) no-repeat center;
  margin-right: 4px;
```

can be a bit simplifier